src/server/vnparse.c

```diff
@@ -2931,11 +3239,92 @@ add_to_vnl(vnl_t **vnlp, char *noden, char *keyw, char *keyval)
 		free(msgbuf);
 		return (1);
 	}
-
+#endif
 	return (0);
 }
 
+#if !(defined(PBS_MOM) || defined(PBS_PYTHON))
 /** 
+ * @brief
+ *	Check if other resources in 'have' satisfy the remaining
+ *	 resources from 'need'.
+ *
+ * @param[in]	need - the need value of resc_limit_t type.
+ * @param[in]	have - the have value of resc_limit_t type.
+ *
+ * @return int
+ *	1 - if not satisfied
+ *	0 - otherwise
+ */
+static int
+check_other_res(resc_limit_t *need, resc_limit_t *have)
```

Comment:
Check for NULL-ness of 'need' and 'have'.

Reply:
Looking at the current invocation, Its impossible to get a NULL value in both parameters. Let me know if you still feel need a redundant check?

Reply:
Yes, please still do the NULL-ness check.

Reply:
Done, see commit https://github.com/PBSPro/pbspro/pull/1452/commits/6151018baa0b74a7f013bf2e272e4ce090374365